### PR TITLE
[WIP - FEEDBACK REQUIRED] Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by m4v15**
relates #7 

I've made a start on the contributing file, thoughts please (@mattlub, @shiryz, @Karyum @sofer )

### CODE REVIEW

**Review by yvonne-liu — CHANGES_REQUESTED** [review]
`space` is showing up as `%20` also it's @m4v15 :wink:

**Review by yvonne-liu — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+Advice for labeling your recipes:
+
+  - Indicate what meal your recipe might apply to (breakfast, lunch or dinner) - it can apply to more than one.
+  - Indicate whether or not the meal is vegetarian friendly
+  - Add any other labels you feel might be appropriate.
+
+## Recipe Structure
+Use the following skeleton guide
+```
+### Name of Dish Here
```
Comment (Karyum) [inline CONTRIBUTING.md:29]: The markdown for this part is not working. (click view file )

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+##### Preparation
+
+1. Add step by step instructions here
+```
+Add photo(s) below preparation steps if possible
+
+### Badges
+
+All recipes should have the following badges: [Difficulty](#difficulty), [Speed](#speed), [Reception](#reception), [Cost](#cost), [Vegetarian](#vegetarian), [Vegan](#vegan)
```
Comment (Karyum) [inline CONTRIBUTING.md:50]: # YES! :tada: :tada:

**Review by Karyum — COMMENTED** [review]
This is some next level shit. :+1:

File: CONTRIBUTING.md (hunk 2 above)
Comment (yvonne-liu) [reply CONTRIBUTING.md:50]: We're obviously ~bored~ being productive.

Comment (Karyum) [reply CONTRIBUTING.md:50]: :laughing: :laughing:

File: CONTRIBUTING.md (hunk 1 above)
Comment (m4v15) [reply CONTRIBUTING.md:29]: It's supposed to be raw markdown there, so people can copy and paste it, then edit it.

Comment (Jbarget) [inline CONTRIBUTING.md]: you can set an issue template @m4v15:

https://help.github.com/articles/creating-an-issue-template-for-your-repository/

**Review by Jbarget — APPROVED** [review]

**Review by Karyum — APPROVED** [review]

**Comment by m4v15** [comment]
Oi @Karyum WIP, feedback required

**Pull request merged**